database/src/migrations/20210726170005_remove_project_region.ts

```diff
@@ -0,0 +1,72 @@
+import * as Knex from 'knex';
```

Comment:
sorry, not sure what this migration file is for... is this not part of the base install?

Reply:
It looks like Shreyas edited the existing migrations to remove all of the foreign key references to the region table, and then dropped region table itself in this new migration.

Reply:
But, we could move this change into the existing migration, if you prefer.  So we just edit the main SQL file to not create the region table at all, and then this new migration file could be removed.